package.json

```diff
@@ -105,37 +105,36 @@
   },
   "dependencies": {
     "@typed-mxgraph/typed-mxgraph": "~1.0.8",
-    "es-toolkit": "~1.16.0",
+    "es-toolkit": "~1.24.0",
     "fast-xml-parser": "4.5.0",
     "mxgraph": "4.2.2",
     "strnum": "1.0.5"
   },
   "devDependencies": {
     "@asciidoctor/core": "~3.0.4",
+    "@eslint/js": "^9.12.0",
```

Comment:
**thought:** we should review the dependabot configuration of the eslint group to ensure that all dependencies are part of configuration